react/src/Root/App/DistrictOverview/index.tsx

```diff
@@ -437,9 +454,9 @@ class DistrictOverview extends React.PureComponent<MyProps, State> {
             && isNotDefined(selectedPLRelocationPointId);
 
         const {
-            mapState,
-            catPoints,
-            relocationSites,
+            mapState: unselectedMapState,
+            catPoints = [],
```

Comment:
Every time catPoints is not defined it will set it as a new emptyList ref and the function below is recalled even if memoized.

Reply:
Also, @tnagorra made it so that it will return catPoints and reloactionSites as empty array by default.